src/main/java/com/sourcery/gymapp/backend/workout/dto/CreateRoutineExerciseDto.java

```diff
@@ -0,0 +1,32 @@
+package com.sourcery.gymapp.backend.workout.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+public record CreateRoutineExerciseDto(
+
+        @NotBlank
+        UUID exerciseId,
+
+        @NotBlank
```

Comment:
The @NotBlank annotation in Java's Bean Validation API (commonly used with Spring Boot) is specifically designed for String fields. It ensures that a string is not null, empty, or composed only of whitespace.
For other types, like Integer, @NotBlank is not appropriate.

If you need to check for specific conditions (e.g., an integer not being zero or negative), you can combine @NotNull with annotations like @Min, @Max, or create a custom validator.